contrib/packs/actions/install.meta.yaml

```diff
@@ -17,6 +18,10 @@
       type: "string"
       default: "actions,aliases,sensors,triggers"
       description: "Possible options are all, triggers, sensors, actions, rules, aliases."
+    upgrade:
+      type: "boolean"
+      default: false
+      description: "Upgrade the pack if already installed."
```

Comment:
This is a bigger discussion that applies to package management in general. It will be idempotent if you specify a precise version, and we already support that, but a repeated `install` is normally regarded as upgrade, we're certainly not alone here. :)